<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="IT1606511" name="com_hackathon_test.CabStationTest" tests="9" failures="0" timestamp="31 Oct 2020 12:51:07 GMT" time="11.254" errors="9">
  <testcase name="testScenarioOne" time="1.026" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.036" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="2.985" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.072" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.018" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.039" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.031" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.015" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
  <testcase name="testScenarioOne" time="1.032" classname="com_hackathon_test.CabStationTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com_hackathon_base.baseUI.takeScreenShotOnFailure(baseUI.java:435)
at com_hackathon_base.baseUI.reportFail(baseUI.java:424)
at com_hackathon_base.baseUI.invokeBrowser(baseUI.java:101)
at com_hackathon_test.CabStationTest.testScenarioOne(CabStationTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testScenarioOne -->
</testsuite> <!-- com_hackathon_test.CabStationTest -->
